### DISCUSSION TITLE
Access local iPad file system in Code

### DISCUSSION THREAD

**Discussion by rcarmo** [blinksh/blink#1510] [Q&A] [answered]
I would like to be able to use Code to edit files in other sandboxed apps like Working Copy. Right now it only lets me edit local files inside its own sandbox, which is not ideal.

I've been using a-Shell, which lets me pick and bookmark folders from other applications and work inside them.

**Comment by carloscabanero** [comment, marked as answer]
Yep, we do that too, have you tried the `link-files` command? It is less than ideal though as with restarts you would have to configure it again, but it will do the trick until we release our git.

**Comment by rcarmo** [comment]
Wow. Where is that documented? I was looking through the Code section and didn’t see any of that…

**Comment by carloscabanero** [comment]
I will make a note so we reference this at the Code documentation.
